features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/telemetry/internal/TelemetryEventHandler.kt

```diff
@@ -38,7 +38,8 @@ internal class TelemetryEventHandler(
 
     private var trackNetworkRequests = false
 
-    private val seenInCurrentSession = mutableSetOf<TelemetryEventId>()
+    private val eventIDsSeenInCurrentSession = mutableSetOf<TelemetryEventId>()
```

Comment:
we might introduce a memory issue here ? How big this set can grow ?

Reply:
We already had this - I've renamed it to make it more clear what is being stored. The size depends on the number of unique telemetry IDs we're sending